lib/cc/cli/runner.rb

```diff
@@ -8,8 +8,8 @@ def self.run(argv)
         new(argv).run
       rescue => ex
         $stderr.puts("error: (#{ex.class}) #{ex.message}")
-
         CLI.debug("backtrace: #{ex.backtrace.join("\n\t")}")
+        exit 1
```

Comment:
I commented when I added this: it was motivated by the work here, but is its own thing, so I figured same PR separate commit was appropriate. It seems to be (somehow) triggering these JUnit reporter errors though so I'm going to pull it out. Baffling.